### PR TITLE
Feature/update coil

### PR DESCRIPTION
**Pull request by omooooori**
I updated the coil version to 1.1.0.

Succeeded to build on Android Studio 4.1.1.
Can open app normally.

### CODE REVIEW

**Comment by uhooi** [comment]
Thanks PR !!
close #70

File: app/build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ android {
 }
 
 dependencies {
-    def coil_version = '0.13.0'
+    def coil_version = '1.1.0'
```
Comment (uhooi) [inline app/build.gradle:58]: Please bump to `1.1.1` .
ref: https://github.com/coil-kt/coil/releases/tag/1.1.1

```suggestion
    def coil_version = '1.1.1'
```

**Review by uhooi — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (omooooori) [reply app/build.gradle]: Sorry for missing v1.1.1.
I changed and committed.

**Comment by uhooi** [comment]
LGTM 👍

**Pull request merged**